tests/devapps/XForms/XForms/App.xaml.cs

```diff
@@ -42,7 +42,7 @@ public partial class App : Application
         public const string B2cClientId = "e3b9ad76-9763-4827-b088-80c7a7888f79";
 
         public static string RedirectUriOnAndroid = Microsoft.Identity.Client.Core.Constants.DefaultRedirectUri; // will not work with system browser
-        public static string RedirectUriOnIos = "adaliosxformsapp://com.yourcompany.xformsapp";
+        public static string RedirectUriOnIos = Microsoft.Identity.Client.Core.Constants.DefaultRedirectUri;
```

Comment:
could you leave the current redirect uri as a comment, this will make it easier for manual testing w/system browser. could you add in android's as well? will save us some time. thx